src/core/Global.ttslua

```diff
@@ -1784,6 +1784,16 @@ function applyOptionPanelChange(id, state)
   elseif id == "enableCardHelpers" then
     toggleCardHelpers(state)
 
+    -- option: Show '1' on damage / horror
+  elseif id == "showOneDamageHorror" then
+    for _, obj in ipairs(getObjects()) do
+      local memo = obj.getMemo()
+      if memo == "damage" or memo == "horror" then
+        local func = obj.getVar("updateLabel")
+        if func then obj.call("updateLabel") end
+      end
+    end
```

Comment:
Thought about adding "Damage" / "Horror" as tag to the tokens, but I think for now this is fine. Might want to revisit this if we need more such calls.